### ISSUE TITLE
Fixed-edges demo should have info explaining what it does

### ISSUE THREAD

**Issue by BeksOmega** [rachel-fenichel/blockly-samples#1070]
**Category**

 - Examples

**Component**

fixed-edges

**Describe the bug**

The fixed edges demo should have some text (like keyboard navigation) to indicate what it is doing. 

**To Reproduce**

1. Go to the fixed-edges demo
2. Observe how there is no info about what is being demo-ed.

**Additional context**

N/A
